gateway/Gopkg.toml

```diff
@@ -12,7 +12,8 @@
 
 [[constraint]]
   name = "github.com/openfaas/nats-queue-worker"
-  version = "0.8.0"
+  source = "git@github.com:bmcstdio/nats-queue-worker"
+  revision = "38c9f291a005379698381680257fd351e7529240"
```

Comment:
Requires https://github.com/openfaas/nats-queue-worker/pull/71 to be merged before a commit/tag from `openfaas/nats-queue-worker` can be used.

Reply:
That has now been merged for you.

Reply:
@alexellis great—I've updated my branch accordingly!